extendr-macros/src/lib.rs

```diff
@@ -710,6 +719,9 @@ pub fn extendr_module(item: TokenStream) -> TokenStream {
                 hidden: true,
             });
 
+            #( functions.extend(#usemetanames().functions); )*
+            #( impls.extend(#usemetanames().impls); )*
```

Comment:
Maybe move these two lines up to where the module functions and implementations are added? (I.e., above the metadata access function.)